app/javascript/components/rooms/room/presentation/Presentation.jsx

```diff
@@ -31,14 +31,14 @@ import { PRESENTATION_MAX_FILE_COEFF, PRESENTATION_SUPPORTED_EXTENSIONS } from '
 export default function Presentation() {
   const { t } = useTranslation();
   const { friendlyId } = useParams();
-  const { data: room } = useRoom(friendlyId);
+  const { data: room, isLoading } = useRoom(friendlyId);
   const { onSubmit } = useUploadPresentation(friendlyId);
 
   const onDrop = (files) => {
     onSubmit(files[0]);
   };
 
-  if (!room.presentation_name) {
+  if (isLoading || !room.presentation_name) {
```

Comment:
Whats this extra check for?

Reply:
When I changed the landing page for the room to `presentation`, an error was being thrown saying that `room` was undefined. Making sure it's loaded first fixed that.

Reply:
You can just do `!room?.presentation_name`

Reply:
Adding the safe operator `?` after every attribute solves many errors